examples/README.md

```diff
@@ -35,5 +35,5 @@ To get started with these examples, please follow the [Quickstart](https://nvfla
    * Illustrates the use of differential privacy for training brain tumor segmentation models using federated learning.
 * [Federated Learning for Prostate Segmentation from Multi-source Data](./prostate/README.md)
   * Example of training a multi-institutional prostate segmentation model using [FedAvg](https://arxiv.org/abs/1602.05629), [FedProx](https://arxiv.org/abs/1812.06127), and [Ditto](https://arxiv.org/abs/2012.04221).
-* [Federated Analysis](./federated_analysis/README.md)
+* [Federated Analysis](federated_statistics/image_stats/README.md)
```

Comment:
Should we rename "Federated Analysis" to "Federated Statistics for Medical Decathlon Data"?

Reply:
Or is it COVID-19 data? I am a bit confusing here